tests/trilinos/vector_reinit_to_linear_map.output

```diff
@@ -0,0 +1,2 @@
+
+DEAL::OK
```

Comment:
It would probably make sense to run with at least two MPI processes. This requires you to change the file name to `vector_reinit_to_linear_map.mpirun=2.output`.

Reply:
Please rename the file to `vector_reinit_to_linear_map.mpirun=2.output`.

Reply:
I accidentally renamed the source file. I'll fix that.